### PR TITLE
feat(*): Add correct answer in chat

### PR DESCRIPTION
**Pull request by dhruvd-grappus**
Logic to generate a random word on server, then show a different state in chat if the user answers correctly.

See video: https://drive.google.com/file/d/1KtNpn9GN1Awd9z8Yszgxw40BJz_RDCIM/view?usp=sharing

### CODE REVIEW

File: api/lib/session/bloc/session_bloc.dart [hunk 1]
```diff
@@ -21,10 +22,15 @@ class SessionBloc extends BroadcastBloc<SessionEvent, SessionState> {
   }
 
   void _onPlayerAdded(OnPlayerAdded event, Emitter<SessionState> emit) {
+    // Round has started when the first player joins the game
+    if (state.players.isEmpty) {
+      emit(state.copyWith(correctAnswer: getRandomWord));
```
Comment (noorbhatia10) [inline api/lib/session/bloc/session_bloc.dart:27]: I think we should start the round when atleast two players have joined. Or we can have a start button, what say?

Comment (noorbhatia10) [reply api/lib/session/bloc/session_bloc.dart:27]: Also, the correct answer is changing when a new player joins

Comment (noorbhatia10) [reply api/lib/session/bloc/session_bloc.dart:27]: Let's make this case **_insensitive_** (uppercase,lowercase)

**Review by noorbhatia10 — COMMENTED** [review]
nice work

File: api/lib/session/bloc/session_bloc.dart (hunk 1 above)
Comment (gauravkatvaa) [reply api/lib/session/bloc/session_bloc.dart:27]: lets consider this, the first player who joined make it admin of the particular sessio. Who can start the session after all the players has joined the game

Comment (dhruvd-grappus) [reply api/lib/session/bloc/session_bloc.dart:27]: for now, the round autostarts when 2 players have joined,
Will target "Start round" feature in next PR.

**Review by noorbhatia — APPROVED** [review]

**Pull request merged**